packages/testing/src/playwright/setup.ts

```diff
@@ -0,0 +1,94 @@
+import { parsePublishableKey } from '@clerk/shared/keys';
+import dotenv from 'dotenv';
+
+type ClerkSetupParams = {
+  /*
+   * The publishable key for your Clerk dev instance.
+   * If not provided, the library will look for the key in the following environment variables:
+   * - NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY
+   * - VITE_CLERK_PUBLISHABLE_KEY
+   * - CLERK_PUBLISHABLE_KEY
+   * - GATSBY_CLERK_PUBLISHABLE_KEY
+   * - REACT_APP_CLERK_PUBLISHABLE_KEY
+   * - EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY
+   */
+  publishableKey?: string;
+  /*
+   * The frontend API URL for your Clerk dev instance, without the protocol.
+   * If provided, it overrides the Frontend API URL parsed from the publishable key.
+   * Example: 'relieved-chamois-66.accounts.dev'
+   */
+  frontendApiUrl?: string;
+  /*
+   * Enable debug mode.
+   */
+  debug?: boolean;
+};
+
+const TESTING_TOKEN_API_URL = 'https://api.clerk.com/v1/testing_tokens';
+
+export const clerkSetup = async (options?: ClerkSetupParams) => {
+  const log = (msg: string) => {
+    if (options?.debug) {
+      console.log(`Clerk: ${msg}`);
+    }
+  };
+
+  log('Setting up Clerk...');
+  dotenv.config({ path: ['.env.local', '.env'] });
+
+  const pubKey =
+    options?.publishableKey ||
+    process.env.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY ||
+    process.env.VITE_CLERK_PUBLISHABLE_KEY ||
+    process.env.CLERK_PUBLISHABLE_KEY ||
+    process.env.GATSBY_CLERK_PUBLISHABLE_KEY ||
+    process.env.REACT_APP_CLERK_PUBLISHABLE_KEY ||
+    process.env.EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY;
+
+  const secretKey = process.env.CLERK_SECRET_KEY;
+  let testingToken = process.env.CLERK_TESTING_TOKEN;
+
+  if (!pubKey) {
```

Comment:
🙃 Do we use this terminology elsewhere? Asking because "pubKey" is a well-established term in crypto, denoting the public key portion of a keypair. We might want to use `publishableKey` to avoid confusion.